examples/cms-notion/components/cover-image.js

```diff
@@ -0,0 +1,25 @@
+import cn from 'classnames'
+import Link from 'next/link'
+
+export default function CoverImage({ title, url, slug }) {
+  const image = (
+    <img
```

Comment:
Could you start with one of the CMS examples that uses `next/image` or update this?